packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/StepOnSuccessVisitor.ts

```diff
@@ -0,0 +1,28 @@
+import stampit from 'stampit';
```

Comment:
should be located in `workflows-1/step/OnSuccessVisitor.ts` and the name of visitor should be `OnSuccessVisitor`.